packages/core/expressions/README.md

```diff
@@ -58,9 +60,10 @@ Import the component(s) in your host application as well as the package styles:
 ```ts
 import { asyncInit, ExpressionsEditor } from '@kong-ui-public/expressions'
 import '@kong-ui-public/expressions/dist/style.css'
+import '@kong-ui-public/forms/dist/style.css' // optional: required for `RouterPlaygroundModal` component
```

Comment:
nit: the host app will also need to register the VFG component via code like:

```js
app.component('VueFormGenerator', VueFormGenerator)
```

Otherwise, they would see a warning like:

<img width="708" src="https://github.com/user-attachments/assets/65df9853-e24b-44b2-aea5-6c77290dadd9">

Reply:
fff

Reply:
Will be also added in KM-391.

Reply:
added